frontend/packages/console-dynamic-plugin-sdk/package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "main": "src/index.ts",
   "scripts": {
-    "clean": "rm -rf dist generated schema",
+    "clean": "rm -rf dist generated",
```

Comment:
The `schema` directory was removed in favor of `generated/schema` some time ago, this is just a cleanup.